examples/connectivity/plot_mne_inverse_label_connectivity.py

```diff
@@ -49,6 +56,27 @@
                     baseline=(None, 0), reject=dict(mag=4e-12, grad=4000e-13,
                                                     eog=150e-6))
 
+###############################################################################
+# Compute inverse solutions and their connectivity
+# ------------------------------------------------
+#
+# Next, we need to compute the inverse solution for this data. This will return
+# the sources / source activity that we'll use in computing connectivity.
+#
+# Now we are ready to compute the connectivity in the alpha band. Notice
+# from the status messages how mne-python:
+#
+# 1. reads an epoch from the raw file
+# 2. applies SSP and baseline correction
+# 3. computes the inverse to obtain a source estimate
+# 4. averages the source estimate to obtain a time series for each label
+# 5. includes the label time series in the connectivity computation
+# 6. moves to the next epoch.
+#
+# This behaviour is because we are using generators, which allows us to
+# compute connectivity in a computationally efficient manner where the amount
+# of memory (RAM) needed is independent from the number of epochs.
```

Comment:
maybe add something like " - because we are using only one epoch at a time." for clarity.